protractor-typescript-cucumber/Config/environment.ts

```diff
@@ -0,0 +1,19 @@
+export class Environment {
```

Comment:
I think it would be best to keep this as simple as possible. Creating an object of environment variables is fine. Creating a class + constructor + new object feels like we are adding a lot of extra code.